cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/inmemory/InMemoryStreamAdmin.java

```diff
@@ -41,8 +41,8 @@ public InMemoryStreamAdmin(InMemoryQueueService queueService) {
   }
 
   @Override
-  public void dropAll() throws Exception {
```

Comment:
I do not think we're doing unrecoverable reset, at least not for 2.8.0. Just delete namespace, which is also unrecoverable. So it would be good. Maybe send a note to folks just to confirm?